src/rpc/blockchain.cpp

```diff
@@ -1220,7 +1220,7 @@ static void BIP9SoftForkDescPushBack(UniValue& softforks, const std::string &nam
     if (consensusParams.vDeployments[id].nTimeout <= 1230768000) return;
 
     UniValue bip9(UniValue::VOBJ);
-    const ThresholdState thresholdState = VersionBitsTipState(consensusParams, id);
+    const ThresholdState thresholdState = VersionBitsState(::ChainActive().Tip(), consensusParams, id, versionbitscache);
```

Comment:
Maybe add a friendly comment in commit about the fact that `versionbitscache` is currently a global and thus the stays the same. Though we can't assert it ?